cmd/wego/gitops/cmd.go

```diff
@@ -75,11 +78,25 @@ func installRunCmd(cmd *cobra.Command, args []string) error {
 	fluxClient := flux.New(cliRunner)
 	kubeClient := kube.New(cliRunner)
 
+	var gitProviderToken string
+	gitProviderToken = os.Getenv("GITHUB_TOKEN")
+	if gitProviderToken == "" {
+		fmt.Print("Github token will be used to create deployment keys and pull requests: ")
```

Comment:
```suggestion
		fmt.Print("Github token will be used to create deploy keys and pull requests: ")
```
"Deploy Keys" is a specific thing in GitHub.

Reply:
Also should this be `Println`?